salt/kubelet/kubelet.jinja

```diff
@@ -7,51 +7,46 @@
 # kubernetes kubelet (minion) config
 
 # The address for the info server to serve on (set to 0.0.0.0 or "" for all interfaces)
-KUBELET_ADDRESS="--address=0.0.0.0"
+# DEPRECATED!!! This option was moved to kubelet-config.yaml
+KUBELET_ADDRESS=""
 
 # The port for the info server to serve on
-KUBELET_PORT="--port={{ pillar['kubelet']['port'] }}"
+# DEPRECATED!!! This option was moved to kubelet-config.yaml
+KUBELET_PORT=""
 
 # Ensure we match the machine hostname
 KUBELET_HOSTNAME="--hostname-override={{ grains['nodename'] }}"
 
 # Add your own!
 KUBELET_ARGS="\
-    --cgroups-per-qos \
-    --cgroup-driver=cgroupfs \
-    --cgroup-root=/ \
-    --kube-reserved-cgroup=podruntime.slice \
-{% if salt.caasp_pillar.get_kubelet_reserved_resources('kube') -%}
+    --config=/etc/kubernetes/kubelet-config.yaml \
+{%- if salt.caasp_pillar.get_kubelet_reserved_resources('kube') %}
     --kube-reserved={{ salt.caasp_pillar.get_kubelet_reserved_resources('kube') }} \
-{% endif -%}
-    --system-reserved-cgroup=system \
-{% if salt.caasp_pillar.get_kubelet_reserved_resources('system') -%}
+{%- endif %}
+{%- if salt.caasp_pillar.get_kubelet_reserved_resources('system') %}
     --kube-reserved={{ salt.caasp_pillar.get_kubelet_reserved_resources('system') }} \
```

Comment:
I think these `kube-reserved` flags could ve moved to the YAML file (as [this](https://github.com/kubernetes/kubernetes/blob/986ed675ac5efcc7696f09d96f314495b1650010/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go#L417) is part of the struct)

Reply:
Ok, I will move it in another PR, since these flags are not deprecated yet